packages/i18n/src/locales/en.i18n.json

```diff
@@ -869,7 +869,7 @@
   "Call_again": "Call again",
   "Call_back": "Call back",
   "Call_not_found": "Call not found",
-  "Call_not_found_error": "This could happen when the call URL is not valid, or you're having connection issues. Please check with the source of the call URL and try again, or talk to your workspace administrator if the problem persists",
+  "Call_not_found_error": "This could happen when the call URL is not valid, or you'r having connection issues. Please check with the source of the call URL and try again, or talk to your workspace administrator if the problem persists",
```

Comment:
```suggestion
  "Call_not_found_error": "This could happen when the call URL is not valid, or you're having connection issues. Please check with the source of the call URL and try again, or talk to your workspace administrator if the problem persists",
```